src/CodeFile.php

```diff
@@ -63,8 +63,6 @@ class CodeFile extends BaseObject
      */
     public function __construct($path, $content, $config = [])
     {
-        parent::__construct($config);
-
```

Comment:
BaseObject doesn't have a constructor anymore, so this was failing.
Is it okay to still use the constructor here ? (I'm really not understanding the DI refactoring 😅)

Reply:
In this case you can just remove `$config` from arguments, AFAICS it was not used actually.